config/m4/ib.m4

```diff
@@ -61,6 +63,8 @@ AS_IF([test "x$with_ib" == xyes],
         save_LDFLAGS="$LDFLAGS"
         save_CFLAGS="$CFLAGS"
         save_CPPFLAGS="$CPPFLAGS"
+        CPPFLAGS="-I$with_verbs/include $CPPFLAGS"
+        LDFLAGS="-L$with_verbs/lib64 -L$with_verbs/lib $LDFLAGS"
```

Comment:
looks good to me, do we need a similar fix below?
BTW have you actually tested it with nonstandard path?